spaceship/lib/spaceship/tunes/members.rb

```diff
@@ -15,7 +15,7 @@ def all
 
         def find(email)
           all.each do |member|
-            if member.email_address == email
+            if member.email_address.downcase == email
```

Comment:
```suggestion
            if member.email_address casecmp?(email)
```

Can we do this instead? ☝️ `email` might not get passed in as all lowercase so we want to make sure its fully caseless compare